.github/prompts/phases/phase-1-issue-creation-prompt.md

```diff
@@ -0,0 +1,201 @@
+- Auto-detect semantic labels based on task wording.
+
+## Completion Enforcement (Post-Run Audit)
+
+After processing the full task list, perform a final audit for every created or updated issue:
+
+- Verify the following fields are populated: Epic Link, Story Points, Assignee, and Original Estimate (timetracking.originalEstimate).
+- If any are missing, immediately call the Jira REST API to set only the missing fields via PATCH, using the dynamically discovered custom field IDs (see `.github/prompts/jira-enforcement-template.md`).
+  - Endpoint: `${JIRA_BASE_URL}/rest/api/3/issue/<ISSUE_KEY>`
+  - Minimal diff payload example (replace placeholders with discovered IDs/values; do not send literal `${...}`):
+
+```json
+{
+  "fields": {
+    "labels": ["PHASE-1"],
+    "${EPIC_LINK_FIELD_ID}": "DEV-739",
+    "${STORY_POINTS_FIELD_ID}": {{STORY_POINTS_NUMBER}},
+    "assignee": { "id": "{{ASSIGNEE_ACCOUNT_ID}}" }
+  }
+}
+```
+
+For timetracking updates, use the edit format:
+
+```json
+{
+  "update": {
+    "timetracking": [{
+      "edit": {
+        "originalEstimate": "{{ORIGINAL_ESTIMATE}}",
+        "remainingEstimate": "{{REMAINING_ESTIMATE}}"
+      }
+    }]
+  }
+}
+```
+
+- Re-validate after patching. Report any remaining deficiencies in the output report and fail the run if mandatory fields are still missing.
+
+## Final Step
+
+Return direct browseable links to each issue: `${JIRA_BASE_URL}/browse/<ISSUE_KEY>`.
+
+## Execution Directive
+
+Perform all feasible automated Jira and Git operations directly; only describe steps that cannot be programmatically completed.
+
+---
+
+## Create a Phase 1 Start Report
+
+- In `.github/prompts/phases`, create file: `phase-1-start-report.md`.
+- Describe initial actions taken to support all tasks within the phase.
+- Include a table in task-number order (order displayed in roadmap) with columns:
+  - Name and description of task
+  - Started date
+  - Status
+  - End date
+  - Time estimate
+  - Points estimate
+  - Link(s) to any supporting context for the implementation agents
+
+---
+
+## Templated Variables
+
+Provide these variables when invoking this template (defaults shown where applicable):
+
+- `PHASE_NUMBER` (required): e.g., `0`, `1`, `2`.
+- `PHASE_NAME` (required): the phase heading suffix exactly as in the roadmap, e.g., `Data Fetch (Indexer Task)`.
+- `ROADMAP_PATH` (default: `.github/requirements/roadmap.md`).
+- `JIRA_PROJECT_KEY` (default: `DEV`).
+- `PARENT_EPIC_KEY` (e.g., `DEV-739`).
+- `PHASE_LABEL` (default: `PHASE-1`).
+- `SUMMARY_PREFIX` (default: `CAA-P1 -`).
+- `DUE_DATE_OFFSET_DAYS` (default: `1`).
+- `ENFORCEMENT_TEMPLATE_PATH` (default: `.github/prompts/jira-enforcement-template.md`).
+
+## Notes
+
+- Ground planning and acceptance criteria in `.github/requirements/index.md` and the referenced slice scope where applicable.
+- Keep diffs small and commits logically grouped per issue branch.
```

Comment:
yet ensure commit messages are descriptive in name and the description outlines all changes and why.  the title must always reference the jira ticket by starting with the key.